library/Falcon/Handler/SES.php

```diff
@@ -0,0 +1,287 @@
+<?php
+
+class Falcon_Handler_SES extends Falcon_Handler_WPMail {
+	/**
+	 * Get a human-readable name for the SES handler
+	 *
+	 * @return string
+	 */
+	public static function get_name() {
+		return __( 'Amazon SES', 'falcon' );
+	}
+
+	/**
+	 * Register handler-specific option fields
+	 *
+	 * @see bbSubscriptions_Handler::register_option_fields
+	 * @param string $group Settings group (4th parameter to `add_settings_field`)
+	 * @param string $section Settings section (5th parameter to `add_settings_field`)
+	 * @param array $options Current options
+	 */
+	public static function register_option_fields($group, $section, $options) {
+		add_settings_field(
+			'bbsub_ses_outbound_status',
+			__( 'Outbound Status', 'falcon' ),
+			[ __CLASS__, 'render_outbound_status' ],
```

Comment:
Should / could this use `static`, wondering if `__CLASS__` is akin to `self`, so subclasses won't be able to override the `render_outbound_status` method.

Reply:
`get_called_class()` would be the one to use, but I'm following the pattern used throughout the rest of Falcon here; this can wait for a refactor I think.